Jenkinsfile

```diff
@@ -168,6 +168,24 @@ try {
         }
       }
     },
+    'CPU: Clang': {
+      node('mxnetlinux-cpu') {
+        ws('workspace/build-cpu-clang') {
+          init_git()
+          def flag = """ \
+            USE_PROFILER=1                \
+            USE_CPP_PACKAGE=1             \
+            USE_BLAS=openblas             \
+            USE_OPENMP=0                  \
+            CXX=clang++                   \
+            CC=clang                      \
+            -j\$(nproc)
+            """
+          make("cpu.clang", flag)
+          pack_lib('cpu.clang')
```

Comment:
Could you replace the dot by an underscore as delimiter to fit the overall style (both lines)?

Reply:
Good catch, will do.